perses/app/relative_point_mutation_setup.py

```diff
@@ -582,6 +607,5 @@ def _solvate(self,
 
         # Canonicalize the solvated positions: turn tuples into np.array
         solvated_positions = unit.quantity.Quantity(value=np.array([list(atom_pos) for atom_pos in solvated_positions.value_in_unit_system(unit.md_unit_system)]), unit=unit.nanometers)
-        solvated_system = self.system_generator.create_system(solvated_topology)
 
-        return solvated_topology, solvated_positions, solvated_system
+        return solvated_topology, solvated_positions
```

Comment:
changing the returnable can be problematic. can you return the solvated system, too?

Reply:
Can you elaborate on why you think changing the returned variables is problematic? This `_solvate()` function is only ever called in the `__init__` of PointMutationExecutor, and I made sure to properly adjust the code in the `__init__` to account for the change in returned variables. Therefore, I don't think its a problem.

I moved the code for creating the system to be outside of the _solvate function (see line 269). This way, if the users specifies a presolvated PDB as input, a system can still be generated without calling _solvate.